src/pipeline/pipeline.ts

```diff
@@ -38,6 +39,8 @@ type Request = {
 	config: BormConfig;
 	tqlRequest?: TQLRequest;
 	dbHandles: DBHandles;
+	// todo: define type
+	enrichedBqlQuery?: any;
```

Comment:
Temporally we can define it as a normal Bql Query, the types should be compatible

Reply:
Also maybe a good timing to package the Request into query and mutation objects, independent one from each other, lets check it together 👀